lms/djangoapps/django_comment_client/management/management_utils.py

```diff
@@ -0,0 +1,49 @@
+'''
+This module provides utility function for the utility functions for django_comment_client
+'''
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+import logging
+from django.db import IntegrityError
+from pymongo.errors import PyMongoError
+
+log = logging.getLogger(__name__)
+
+
+def rename_user(old_username, new_username):
+    '''
+    This function changes the username of a user
+    :param old_username: old username of the user
+    :param new_username: new username desired for this user
+    :return: result {'status': 'success'/'failure', 'message': 'relevant message'}
+    '''
+    result = {'status': 'failure', 'message': ''}
+    try:
+        user = User.objects.get(username=old_username)
+    except User.DoesNotExist:
```

Comment:
I wonder if we can do without this exception handling. The idea here is
that if the `username_old` does not exist, we should halt the operation
and short-circuit return immediately. We'll get this "for free" by just
not even trying the catch the exception; the `User.DoesNotExist` will be
thrown, bubble up to the management command, and write the error to the
screen. This also gives us:
- error text written to STDERR instead of STDOUT
- a non-zero exit code

Let me know what you think.